guides/source/6_0_release_notes.md

```diff
@@ -1001,6 +1001,8 @@ Ruby on Rails Guides
 Please refer to the [Changelog][guides] for detailed changes.
 
 ### Notable changes
+*   Add Multiple Databases with Active Record guide.
```

Comment:
Can we add an empty line between "Notable changes" and this line for consistency with other sections?